src/python/test/help_messages/extract_expression_help_message.txt

```diff
@@ -53,21 +55,25 @@ optional arguments:
                         corresponding to "--retrieve-expression". Must be used
                         with "--retrieve-expression" flag. Either "--filter-
                         json" or "--filter-json-file" may be supplied, not
-                        both.
+                        both. If no filter is provided, an SQL query returning
```

Comment:
`If no filter is provided, an SQL ...`
to
`If no filter is provided, a SQL ...`